.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "sarif-viewer.connectToGithubCodeScanning": "off"
```

Comment:
do you think this should be checked in, or should we .gitignore these settings?  Not sure what this one does.